src/api/src/lib.rs

```diff
@@ -10,7 +10,7 @@ use std::convert::TryFrom;
 use std::result;
 
 use clap::{App, Arg};
-use vmm::{KernelConfig, MemoryConfig, VMMConfig, VcpuConfig};
+use vmm::{BlockConfig, KernelConfig, MemoryConfig, NetConfig, VMMConfig, VcpuConfig};
```

Comment:
I'd remove `NetConfig` from this PR and save it for later.

Reply:
It's not super clean, but the net PR will follow immediately after so it seemed more straightforward to do things like that.